pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -366,6 +428,53 @@ class CreateVM extends React.Component {
                            onChange={this.onChangedEventChecked.bind(this, 'startVm')} />
                     {_("Immediately Start VM")}
                 </label>
+
+                <hr />
+
+                { this.props.providerName == 'LibvirtDBus' &&
+                this.state.sourceType != EXISTING_DISK_IMAGE_SOURCE &&
+                    <React.Fragment>
+                        <label className="checkbox-inline">
+                            <input id="choose-storage" type="checkbox"
+                                   checked={this.state.chooseStorage}
+                                   onChange={this.onChangedEventChecked.bind(this, 'chooseStorage')} />
```

Comment:
We prefer to bind the function in the contructor to avoid binding them at each re-render.

Reply:
Well, I tried to keep constistency, because rest of the component binds functions in-place.
Actually, what is the preferred way in these situations? If I add something to component, which implements stuff "and old way", should I keep consistency and then possibly change it to "new way" in another PR, or should I implement the "the new way" no matter of consistency?

Reply:
If something 'the new way' makes performance better I would go with the new way and no care so much about consistency.